types/route.d.ts

```diff
@@ -0,0 +1,59 @@
+import * as http from 'http'
+import * as http2 from 'http2'
+import * as https from 'https'
+
+import { FastifyInstance } from '../fastify'
+import { FastifyMiddleware, FastifyMiddlewareWithPayload } from './middleware'
+import { FastifyRequest } from './request'
+import { FastifyReply } from './reply'
+import { FastifySchema, FastifySchemaCompiler } from './schema'
+import { HTTPMethods, RawServerBase, RawServerDefault, RawRequestBase, RawRequestDefault, RawReplyBase, RawReplyDefault } from './utils'
+
+export type RouteShorthandMethod<
+  RawServer extends RawServerBase = RawServerDefault, 
+  RawRequest extends RawRequestBase = RawRequestDefault, 
+  RawReply extends RawReplyBase = RawReplyDefault
+> = (path: string, handler: RouteHandlerMethod) => FastifyInstance<RawServer, RawRequest, RawReply>
+
+export type RouteShorthandMethodWithOptions<
```

Comment:
This type is not complete. The handler may also be supplied via the options object. (ref: https://github.com/fastify/fastify/blob/master/docs/Routes.md).
```js
const opts = {
  schema: {
    response: {
      200: {
        type: 'object',
        properties: {
          hello: { type: 'string' }
        }
      }
    }
  },
  handler (request, reply) {
    reply.send({ hello: 'world' })
  }
}
fastify.get('/', opts)
```

Reply:
Thank you! I'll make sure to add this